src/bin/common.rs

```diff
@@ -175,6 +175,13 @@ pub fn parse_cli() -> Result<CliOptions, CliError> {
         .takes_value(true)
         .default_value("6")
     )
+    .arg(
+      Arg::with_name("SCENE_CHANGE_DETECTION_SPEED")
+        .help("Speed level (0 is best quality, 2 is fastest)")
```

Comment:
Might be good if we can say current state in a nice and concise way, 
One idea I had was like this, feel free to edit and modify:) 
```suggestion
        .help("Specify Speed of scene-change detection, 0: best, 1: medium, 2: Fastest mode[default:  0 for s0-s6, 1 for s7-s9, 2 for s10]")
```